README.md

```diff
@@ -1,2 +1,60 @@
 # hac-infra
+
 Hybrid Application Console Infrastructure repository
+
+## Local Development
+
+Since the plugin is not served directly from console.redhat.com you have to run webpack proxy in order to properly develop locally. You have two options
+
+* With hac-core served from CDN - if you want to test your changes just in your plugin
+* With hac-core locally - if you want to add new plugin or change how navigation is treated.
+
+### With hac-core served from CDN (default)
+
+Once your plugin has been enabled in hac-core and promoted to at least stage environment you can start serving hac-core from CDN. You can just run
+
+```
+yarn dev
+```
+
+Note: This will require the internal VPN.
+
+### With hac-core locally (advanced)
+
+In order to run hac-core and your plugin together, you will have to pull https://github.com/openshift/hac-core repository, install all dependencies and follow [dynamic plugins](https://github.com/openshift/hac-core#dynamic-plugins). Once hac-core is running you'll have to run your plugin in federated mode.
+
+```
+yarn dev:federated
+```
+
+### Variables
+
+* BETA - to switch between stable and beta releases
```

Comment:
Maybe would be good to explain the use cases hare, meaning when to set `BETA` and when not?

Reply:
Ah yeah, ConsoleDot-isms... I'll see if I can document this better.